### ISSUE TITLE
Flux Coupling Analysis

### ISSUE THREAD

**Issue by achillesrasquinha** [opencobra/cobrapy#962]
I believe cobrapy doesn't provide FCA out-of-the-box yet. Thought I'd initiate the discussion. Any request for having this? Would love to have a PR for the same.

**Comment by cdiener** [comment]
Yes that would be in scope (see https://github.com/opencobra/cobrapy/wiki/Scope). Did you want to get on this?
